frontend/packages/data-portal/app/components/Table/index.ts

```diff
@@ -1,6 +1,6 @@
+export * from '../TablePageLayout/TableCount'
```

Comment:
Not sure if this is the time for refactoring, but I recently learned about barrel files hurting performance:
https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/

Reply:
interesting. i wonder if this is the reason tooling like ESLint feels so slow lately